docs/sources/architecture/configuration.md

```diff
@@ -29,6 +29,13 @@ Configuration descriptions must be placed as XML file(s) (with the ending `.xml`
         http://eclipse.org/smarthome/schemas/config-description-1.0.0.xsd">
 
   <config-description uri="{binding|thing-type|bridge-type|channel-type|any_other}://bindingID:...">
+    <group name="String">
```

Comment:
I just put one group before the parameters and one after and it passes the tests. I need to update Eclipse to use a local schema so I can validate what the schema thinks...